src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6763,6 +6766,36 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ListPatternRequiresLength" xml:space="preserve">
     <value>List patterns may not be used for a value of type '{0}'. No suitable 'Length' or 'Count' property was found.</value>
   </data>
+  <data name="WRN_UseDefViolationPropertyStructThis" xml:space="preserve">
+    <value>Use of possibly unassigned auto-implemented property '{0}'</value>
```

Comment:
The message seems a bit misleading. The warning is that we had to make a default assignment.
How about "The use of auto-implemented property '{0}' causes an implicit initialization with 'default'. Consider initializing explicitly."?

I'd apply similar wording to the other new warnings.